### DISCUSSION TITLE
md-nav__list is empty once deployed, but has correct left hand navigation locally

### DISCUSSION THREAD

**Discussion by AlliNeuman** [squidfunk/mkdocs-material#6690] [Q&A] [answered]
Good evening,
I am working on my teams doc site and previously it was working with no issues. I updated the navigation of the site and added new files to match. It runs locally and I can see the navigation, but when I deploy it the left side navigation is an empty string.  I have a github actions set up to build the site and am on the most recent mkdocs version and mkdocs-material version. The issue is deployed in our NP environment and the code is basically the same as Prod, though prod does not have this navigation issue. Has anyone seen something like this or have an idea of what is going on? The html file after being built just doesn't include the md-nav__list at all.

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @AlliNeuman,
without a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) it will be hard to pinpoint the issue, but since it works locally then it means that your remote environment doesn't match the local one.
- How do you deploy the site? You mentioned a GitHub Action, so perhaps check the logs in the build for warnings / errors, and clean any caches. Does the workflow install the python files the same you do on your local machine?
- Vice-versa download the deployed files and run them locally or do a `diff` check of the local and deployed files if possible.
- If there are no errors yet the navigation still doesn't exist in the built page, I would next check if there are any overrides of [`site_nav`](https://github.com/squidfunk/mkdocs-material/blob/9d52e36212a6d1c32308a5e81084b81eb6f075f2/src/templates/base.html#L295) block or the [`partials`](https://github.com/squidfunk/mkdocs-material/tree/master/src/templates/partials) that use the `md-nav_list` like `nav.html` etc. Is `toc.html` unaffected?

**Reply by AlliNeuman** [reply]
There was an override file for the nav that I didn't dig into. That was the issue, thank you!
